Marlin/src/pins/stm32g0/pins_BTT_EBB42_V1_1.h

```diff
@@ -0,0 +1,157 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2021 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+/** CAUTION **
+ *This board definition is to facilitate support for a Filament Extrusion
+ *devices, used to convert waste plastic into 3D printable filament. 
+ *This board is NOT a general 3D printing controller,
+ *it is NOT supported as a toolboard via CANBUS (as it was originally designed)
+ *or any device that requires kinematics
+ */
+#pragma once
+
+#ifndef BOARD_INFO_NAME
+  #define BOARD_INFO_NAME "BTT EBB42 V1.1"
+#endif
+
+//
+// EEPROM
+//
+#if EITHER(NO_EEPROM_SELECTED, FLASH_EEPROM_EMULATION)
+  #ifndef FLASH_EEPROM_EMULATION
+    #define FLASH_EEPROM_EMULATION
+  #endif
+  #define EEPROM_PAGE_SIZE      (0x800UL) // 2K
+  #define EEPROM_START_ADDRESS      (0x8000000UL + (FLASH_SIZE) - (EEPROM_PAGE_SIZE))
```

Comment:
`EEPROM_START_ADDRESS` is usually defined as the following in other STM32 pins files:
```cpp
#define EEPROM_START_ADDRESS (0x8000000UL + (STM32_FLASH_SIZE) * 1024UL - (EEPROM_PAGE_SIZE) * 2UL)
```